generators/app/templates/src/app/components/Checkbox/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState } from 'react';
+
+interface Props {
+  checked?: boolean;
+  className?: string;
+  inputClassName?: string;
+  labelClassName?: string;
+  label?: string | undefined;
+  name?: string;
+  disabled?: boolean;
+  required?: boolean;
+  onChange?: (event: React.FormEvent<Element>) => void;
+}
+
+function CheckboxContainer({
```

Comment:
I'd change the name to Checkbox, now that there's no layout component. And I'd add an `error` prop that we show below the checkbox like we usually do with inputs, and style it in some default way (red color for the text, red border for the checkbox) and then each project will only have to change those default values and they'll have a fully functional Checkbox. What do you think?

Reply:
I like it